src/class_consts.ts

```diff
@@ -7,6 +7,8 @@ import selectionStyles from 'css/selection.scss';
  */
 export const PPViewerIndirectChildClass = 'pp-viewer-indirect-child';
 
+export const PPViewerHoverContainerClass = 'pp-viewer-hover-container';
+
```

Comment:
While we have talked over all the changes, I guess this won't be self-explanatory for any person joining the project, so I created an issue #150 for a global strategy of tracking mouse hover.